reference_entities/enrich_a_record_with_new_attribute_type.rst

```diff
@@ -0,0 +1,443 @@
+Enrich Records with a new Reference Entity Attribute type
+=========================================================
+
+.. note::
+
+   Reference Entities feature is only available for the **Enterprise Edition**.
+
+This cookbook will present how to enrich Records with a custom Reference Entity Attribute Type we just created `in a previous step`_.
+
+.. _in a previous step: ./create_new_attribute_type.html
+
+Enrich Records With the Attribute
+---------------------------------
+
+In the previous tutorial, we've created a custom simple metric attribute.
+In this tutorial, we will be able to enrich this attribute directly in the records of the reference entity, for example here, the Surface attribute:
+
+.. image:: ../_images/reference_entities/enrich_record_simple_metric_attribute.png
+  :alt: Enrich a Simple Metric value on a record
+
+1) Your New Record Value
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enrich a record, we will create a new Record Value for the brand new Attribute type.
```

Comment:
Capital letters seems a bit weird here, "Record Value" but "Attribute type". Maybe we should decide and use always the same notation